cmd/svcat/class/get_cmd.go

```diff
@@ -57,6 +60,13 @@ func NewGetCmd(cxt *command.Context) *cobra.Command {
 		false,
 		"Whether or not to get the class by UUID (the default is by name)",
 	)
+	cmd.Flags().StringVarP(
+		&getCmd.broker,
+		"broker",
+		"b",
+		"",
+		"Display the associated broker.",
```

Comment:
I don't find this to be an accurate description of what this flag does. It does not display the broker. It displays the classes made available by the broker.

Reply:
Suggested flag description "Filters the classes by a broker name"